### PR TITLE
Formatting change for NSMBot PR message

### PR DESCRIPTION
**Pull request by Mixaster995**
Signed-off-by: Mikhail Avramenko <avramenkomihail15@gmail.com>

## Description

First step to make NSMBot messages more human-readable

## Issue link

https://github.com/networkservicemesh/cmd-template/issues/64

## How Has This Been Tested?

- [ ] Added unit testing to cover
- [x] Tested manually
- [ ] Tested by integration testing
- [ ] Have not tested

## Types of changes

- [ ] Bug fix
- [ ] New functionallity
- [ ] Documentation
- [x] Refactoring
- [x] CI

### CODE REVIEW

**Review by Bolodya1997 — APPROVED** [review]

**Comment by Mixaster995** [comment]
Tested on 4 repositories chain
Now messages looks like this:
### Link - https://github.com/Mixaster995/test-actions-4/pull/8
### Image:

Screenshot 2021-07-30 at 12 06 18

**Review by Bolodya1997 — APPROVED** [review]

**Comment by edwarnicke** [comment]
@Mixaster995 Is this ready to merge?

**Comment by Bolodya1997** [comment]
> @Mixaster995 Is this ready to merge?

@edwarnicke If you are OK with these changes, probably we want to wait for some time and add PRs to all other repositories to merge them in the same time.

**Review by denis-tingaikin — APPROVED** [review]
Looks good.

@Mixaster995 Please apply this for all other repos.

File: .github/workflows/update-dependent-repositories-gomod.yaml [hunk 1]
```diff
@@ -36,20 +36,25 @@ jobs:
           path: ${{ github.repository }}
           repository: ${{ github.repository }}
           token: ${{ secrets.NSM_BOT_GITHUB_TOKEN }}
+      - name: Get sha
+        id: get-sha
+        run: |
+          echo ::set-output name=sha::$( curl -u "u:${{secrets.NSM_BOT_GITHUB_TOKEN}}" https://api.github.com/repos/networkservicemesh/sdk/git/ref/heads/main | jq .object.sha | tr -d '"' )
```
Comment (denis-tingaikin) [inline .github/workflows/update-dependent-repositories-gomod.yaml:42]: Please use GITHUB_SHA env. or for context ${{github.sha}}

**Pull request merged**